### PR TITLE
Added clarification to Stream FAQ 

### PR DESCRIPTION
**Pull request by marciocloudflare**
Clarification is about the limit of concurrent videos being uploaded

### CODE REVIEW

File: products/stream/src/content/faq/index.md [hunk 1]
```diff
@@ -18,7 +18,13 @@ You cannot download the *exact* input file that you uploaded. However, depending
 
 - By default, a video upload can be at most 30 GB.
 
-- By default, 120 videos can be in the `inprogress`, `queued` or `downloading` state at once. If videos are in `error`, `ready` or `pendingupload` states, they don't count towards this limit.
+- By default, 120 videos can be in the `inprogress`, `queued` or `downloading` state at once. If videos are in `error`, `ready` or `pendingupload` states, they don't count towards this limit.
```
Comment (pedrosousa) [inline products/stream/src/content/faq/index.md]:
```suggestion
- By default, you can have up to 120 videos in the `inprogress`, `queued` or `downloading` state at the same time. Videos in the `error`, `ready` or `pendingupload` state do not count toward this limit. 
```

File: products/stream/src/content/faq/index.md [hunk 2]
```diff
@@ -18,7 +18,13 @@ You cannot download the *exact* input file that you uploaded. However, depending
 
 - By default, a video upload can be at most 30 GB.
 
-- By default, 120 videos can be in the `inprogress`, `queued` or `downloading` state at once. If videos are in `error`, `ready` or `pendingupload` states, they don't count towards this limit.
+- By default, 120 videos can be in the `inprogress`, `queued` or `downloading` state at once. If videos are in `error`, `ready` or `pendingupload` states, they don't count towards this limit. 
+
+<Aside type="note">
+
+The limit to the number of videos only applies to videos being uploaded to Cloudflare Stream. It is not related to the number of end-users streaming videos.
```
Comment (pedrosousa) [inline products/stream/src/content/faq/index.md]:
```suggestion
The limit to the number of videos only applies to videos being uploaded to Cloudflare Stream. This limit is not related to the number of end users streaming videos.
```

File: products/stream/src/content/faq/index.md (hunk 1 above)
Comment (pedrosousa) [reply products/stream/src/content/faq/index.md]: I believe that "at the same time" expresses it better than "at once".

Comment (pedrosousa) [reply products/stream/src/content/faq/index.md]: "Toward" is more common in American English.

**Review by pedrosousa — APPROVED** [review]

File: products/stream/src/content/faq/index.md [hunk 3]
```diff
@@ -18,7 +18,13 @@ You cannot download the *exact* input file that you uploaded. However, depending
 
 - By default, a video upload can be at most 30 GB.
 
-- By default, 120 videos can be in the `inprogress`, `queued` or `downloading` state at once. If videos are in `error`, `ready` or `pendingupload` states, they don't count towards this limit.
+- By default, you can have up to 120 videos in the `inprogress`, `queued` or `downloading` state at the same time. Videos in the `error`, `ready` or `pendingupload` state do not count toward this limit.
```
Comment (zaidf) [inline products/stream/src/content/faq/index.md:21]: I would add a sentence like "If you need the concurrency limit raised, please write to support (link to support) explaining. your use case and a justification for why you would like the limit raised."

**Pull request merged**